<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtLiaisonSerieClass</class>
 <widget class="QMainWindow" name="QtLiaisonSerieClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtLiaisonSerie</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QComboBox" name="portChoiceComboBox">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>30</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="portLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Port :</string>
    </property>
   </widget>
   <widget class="QLabel" name="portStatusLabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Statut port : ferme</string>
    </property>
   </widget>
   <widget class="QLabel" name="receivedDataLabel">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>100</y>
      <width>541</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>données reçues :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="receivedDataText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>551</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="openPortButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Ouvrir le port</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sendButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>300</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>793</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtLiaisonSerie.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>openPortButton</sender>
   <signal>clicked()</signal>
   <receiver>QtLiaisonSerieClass</receiver>
   <slot>onOpenPortButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendButton</sender>
   <signal>clicked()</signal>
   <receiver>QtLiaisonSerieClass</receiver>
   <slot>onSendButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>525</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>396</x>
     <y>247</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onOpenPortButtonClicked()</slot>
  <slot>onSerialPortReadyRead()</slot>
  <slot>onSendButtonClicked()</slot>
 </slots>
</ui>
